pyvista/core/filters.py

```diff
@@ -1129,6 +1129,24 @@ def warp_by_scalar(dataset, scalars=None, factor=1.0, normal=None,
             return
         return output
 
+    def warp_by_vector(dataset, scalars=None, factor=1.0):
+        """Warp the dataset's points by a point data vectors array's values.
+
+        This modifies point coordinates by moving points along point vectors by
+        the local vector times the scale factor.
+
+        A classical application of this transform is to visualize eigenmodes in mechanics.
+        """
```

Comment:
Please expand the docstring to include `Parameters`, `Returns`, and `Example`.  See the other docstrings for an example.

One note, anything you write in the `Examples` section will be run within the pipeline and have its outputs checked against some expected value (anything after `>>>`).  Make sure if something returns something, it's set to a variable so it doesn't automatically print out.

Reply:
It would be great to have a small example, but I'm not sure what would work here. Do you have any pointers to a simple to setup example which could be applied to the vector warping?